docs/source/devguides/supporting/examples/NodeRed.rst

```diff
@@ -0,0 +1,79 @@
+.. _NodeRed:
+
+Node Red Example
+================
+
+Node Red is a visual programming wherein users connect small units of
+functionality "nodes" to create "flows".
+
+There are two example nodes that allow communication between Node Red and
+Volttron. One node reads subscribes to messages on the VOLTTRON message bus
+and the other
+
+Dependencies
+------------
+
+The example nodes depend on `python-shell` to be installed and available to
+the Node Red environment.
```

Comment:
python-shell is a Node Red dependency. Should it still go in optional_requirements.json?